components/atom/checkbox/src/index.scss

```diff
@@ -24,7 +24,8 @@ $bd-atom-checkbox-is-intermediate: $bd-atom-checkbox--checked !default;
 $bdc-atom-checkbox-is-disabled: color-variation($c-gray, 1) !default;
 $bgc-checked-atom-checkbox-is-disabled: color-variation($c-gray, 3) !default;
 $bgc-checked-atom-checkbox-is-intermediate: $bgc-checked-atom-checkbox !default;
-$c-atom-checkbox-custom-is-disabled: $c-atom-checkbox !default;
+$c-atom-checkbox-is-disabled: $c-gray !default;
+$c-atom-checkbox-custom-is-disabled: $c-atom-checkbox-is-disabled;
 $c-atom-checkbox-is-disabled: color-variation($c-gray, -2) !default;
```

Comment:
L27 & L29 are assigning values to the same token. This is gonna ignore the first assignement.

Reply:
I think the correct value is the L29
`$c-atom-checkbox-is-disabled: color-variation($c-gray, -2) !default;`